Framework/CurveFitting/src/FuncMinimizers/DampedGaussNewtonMinimizer.cpp

```diff
@@ -30,6 +30,7 @@ DECLARE_FUNCMINIMIZER(DampedGaussNewtonMinimizer, Damped GaussNewton)
 DampedGaussNewtonMinimizer::DampedGaussNewtonMinimizer(double relTol)
     : IFuncMinimizer(), m_relTol(relTol) {
   declareProperty("Damping", 0.0, "The damping parameter.");
+  declareProperty("Debug", false, "Turn on the debug output.");
```

Comment:
What you describe sounds awfully lot like debugging, and there is the *Notices* logging level above *Debug* as well, if you get overwhelmed with the amount of messages. Anyway, I don't feel strongly about this. If you think the verbose option will help users to better grasp what happened during the fitting, then I guess the property is justified.